### DISCUSSION TITLE
after "Lights off" lamps remain switched on

### DISCUSSION THREAD

**Discussion by Sebi77ber** [basnijholt/adaptive-lighting#309] [Q&A]
Hello everyone. 

I have problems with the integration. Generally everything works, but the lamps do not turn off reliably with the service "Lights off". It happens again and again that individual lamps remain switched on. On the HA interface, however, these are shown as switched off. Only when I manually switch the lamps on and off again via the user interface, they actually turn off. After I uninstalled the integration, everything works without problems. All the time automations switch off lamps, they are off. So it's not an ZigBee issue I think. 

Turning them on via automations and Adaptive Lightning always work without problems.

Does anyone have any idea what the problem could be?

Thanks for your help!

Regards
Sebastian

**Comment by daFreeMan** [comment]
You're not alone. See issue #343

**Comment by th3w1zard1** [comment]
#450 will fix this.
